packages/backend/server/src/core/doc-service/controller.ts

```diff
@@ -47,9 +47,14 @@ export class DocRpcController {
   @Get('/workspaces/:workspaceId/docs/:docId/markdown')
   async getDocMarkdown(
     @Param('workspaceId') workspaceId: string,
-    @Param('docId') docId: string
+    @Param('docId') docId: string,
+    @Query('aiEditable') aiEditable: boolean
```

Comment:
aiEditable is string，should convert it to bool

```ts
aiEditable === 'true'
```

Reply:
like fullContent does
![image](https://github.com/user-attachments/assets/0e812409-af62-42c5-9e21-ae49f65c4fcb)

Reply:
good point, fixed